src/bodies/minor.jl

```diff
@@ -0,0 +1,55 @@
+export MinorBody, pluto, Pluto, pluto_barycenter, PlutoBarycenter
+
+abstract type MinorBody <: CelestialBody end
+
+const MINOR_BODY_NAMES = (
+    "Ceres",
+    "Pallas",
+    "Vesta",
+    "Lutetia",
+    "Ida",
+    "Eros",
+    "Davida",
+    "Gaspra",
+    "Steins",
+    "Itokawa",
+    "Tempel1",
+    "Borrelly",
+)
+
+for body in MINOR_BODY_NAMES
+    str_sym = Symbol(body)
+    con_sym = Symbol(lowercase(body))
+    @eval begin
+        struct $str_sym <: MinorBody end
+        const $con_sym = $str_sym()
+        parent(::$str_sym) = ssb
+        export $con_sym
+    end
+end
+
+struct PlutoBarycenter <: Barycenter end
+const pluto_barycenter = PlutoBarycenter()
+Base.show(io::IO, ::PlutoBarycenter) = print(io, "Pluto Barycenter")
+parent(::PlutoBarycenter) = ssb
+naifid(::PlutoBarycenter) = 901
+
+struct Pluto <: MinorBody end
+const pluto = Pluto()
+Base.show(io::IO, ::Pluto) = print(io, "Pluto")
```

Comment:
Probably missed this to remove :wink:

Reply:
Would not have happened if Pluto was still a planet 😢